mne/minimum_norm/__init__.py

```diff
@@ -11,3 +11,4 @@
 
 from .resolution_matrix import (make_resolution_matrix, get_point_spread,
                                 get_cross_talk)
+from .resolution_metrics import resolution_metrics
```

Comment:
you should rename resolution_metrics.py to _resolution_metrics.py to avoid problems of function having the same name as the file. Use `git mv resolution_metrics.py _resolution_metrics.py` for this